CMakeLists.txt

```diff
@@ -141,8 +141,8 @@ find_package(ZLIB REQUIRED)
 find_package(Pixman REQUIRED)
 
 # Check for gettext
-option(ENABLE_NLS "Enable translation of program messages" ON)
-if(ENABLE_NLS)
+set(ENABLE_NLS CACHE BOOL "Enable translation of program messages")
+if(ENABLE_NLS OR ENABLE_NLS STREQUAL "")
```

Comment:
I sympathize with the goal here, but I'm concerned that the method gets a bit convoluted. Leaving booleans without a clear value, and doing string comparisons with them, can be a bit confusing and easy to get wrong.

Are you aware of any alternative models to solve this issue?

Reply:
I rewrote the code to use NOT DEFINED instead.